web/src/main/java/org/cbioportal/web/util/LinearDataBinner.java

```diff
@@ -34,12 +34,8 @@
 
     public static final Integer DEFAULT_INTERVAL_COUNT = 20;
 
-    private DataBinHelper dataBinHelper;
-
     @Autowired
-    public LinearDataBinner(DataBinHelper dataBinHelper) {
-        this.dataBinHelper = dataBinHelper;
-    }
+    private DataBinHelper dataBinHelper;
```

Comment:
What am I missing here? You add the dataBinHelper, but never use it.

Reply:
Same for LogScaleDataBinner.java, ScientificSmallDataBinner.java

Reply:
dataBinHelper is used in this class. I just changed to constructor injection to property injection